examples/chapter-list.lua

```diff
@@ -20,32 +41,58 @@ local function open_chapter()
     end
 end
 
---dynamic keybinds to bind when the list is open
-list.keybinds = {
-    {'DOWN', 'scroll_down', function() list:scroll_down() end, {repeatable = true}},
-    {'UP', 'scroll_up', function() list:scroll_up() end, {repeatable = true}},
-    {'ENTER', 'open_chapter', open_chapter, {} },
-    {'ESC', 'close_browser', function() list:close() end, {}}
-}
-
 --update the list when the current chapter changes
 mp.observe_property('chapter', 'number', function(_, curr_chapter)
     list.list = {}
     local chapter_list = mp.get_property_native('chapter-list', {})
     for i = 1, #chapter_list do
         local item = {}
-        if (i-1 == curr_chapter) then
+        if (i - 1 == curr_chapter) then
+            list.selected = curr_chapter + 1
             item.style = [[{\c&H33ff66&}]]
         end
 
         local time = chapter_list[i].time
+        local title = chapter_list[i].title
+        if title == "" then title = "Chapter " .. string.format("%02.f", i) end
         if time < 0 then time = 0
         else time = math.floor(time) end
         item.ass = string.format("[%02d:%02d:%02d]", math.floor(time/60/60), math.floor(time/60)%60, time%60)
-        item.ass = item.ass..'\\h\\h\\h'..list.ass_escape(chapter_list[i].title)
+        item.ass = item.ass..'\\h\\h\\h'..list.ass_escape(title)
         list.list[i] = item
     end
```

Comment:
```suggestion
    end
    
    list.selected = curr_chapter + 1
    list.list[curr_chapter + 1].style = [[{\c&H33ff66&}]]
```

Reply:
There is a problem with the change here, the `curr_chapter` may be a nil value. It is better to keep it unchanged.

Reply:
Ah you're right, we can put it in an if block then: `if curr_chapter then`

Reply:
> Ah you're right, we can put it in an if block then: `if curr_chapter then`

This is fine, but I want to stay in the loop as a better example script. 
Because in some usage scenarios,  the `list.selected` must be set in the loop. You can refer to another script of mine, such as [adevice-list.lua](https://github.com/dyphire/mpv-scripts/blob/main/adevice-list.lua).